src/v/test_utils/randoms.h

```diff
@@ -67,7 +67,7 @@ auto random_tristate(Func f) {
 }
 
 template<typename Fn, typename T = std::invoke_result_t<Fn>>
-inline auto random_vector(Fn&& gen, size_t size = 20) -> std::vector<T> {
+inline auto random_vector(Fn&& gen, size_t size = 2) -> std::vector<T> {
```

Comment:
intentional?

Reply:
yes. though i guess debatable. i think i'll drop it. it made debugging easier